packages/access-api/wrangler.toml

```diff
@@ -28,43 +25,52 @@ database_id = "7c676e0c-b9e7-4711-97c8-7b1c8eb229ae"
 [[r2_buckets]]
 binding = "DELEGATIONS_BUCKET"
 bucket_name = "w3up-delegations-dev-0"
+preview_bucket_name = "w3up-delegations-dev-0"
 
 [vars]
 ENV = "dev"
 DEBUG = "true"
 DID = "did:web:local.web3.storage"
+PRIVATE_KEY="MgCYWjE6vp0cn3amPan2xPO+f6EZ3I+KwuN1w2vx57vpJ9O0Bn4ci4jn8itwc121ujm7lDHkCW24LuKfZwIdmsifVysY="
 UPLOAD_API_URL = "https://up.web3.storage"
 
 [build]
-command = "scripts/cli.js build"
 watch_dir = "src"
 
 [miniflare]
 d1_persist = ".wrangler/miniflare"
 
+rules = [
+  { type = "Text", globs = ["package.json"], fallthrough = true }
+]
```

Comment:
what dis?

Reply:
I believe this was required for cloudflare workers to be able to `import` from the package.json.
I think it's a bundler-independent way of providing enough information about whether importing from that json file should result in a string `Text` or a json-parsed object.
https://developers.cloudflare.com/workers/wrangler/configuration/#bundling

Reply:
I don't think I need it anymore, since now release.node.js imports it in node at build time, and writes a rendered value for cloudflare to import via ESM (so it doesn't need to import the package.json to build the rendered version).

I'll remove it